Assets/BossRoom/Scripts/Client/Game/Character/ClientCharacterVisualization.cs

```diff
@@ -64,14 +64,30 @@ public class ClientCharacterVisualization : NetworkBehaviour
 
         Quaternion m_LerpedRotation;
 
+        void Awake()
+        {
+            foreach (var rendererComponent in GetComponentsInChildren<Renderer>())
```

Comment:
Why not SetActive(false) on the whole GameObject? Why disable visual components only?

Reply:
Because if the GameObject is not active, it will never be spawned